Tests/XCUITests/SaveLoginsTests.swift

```diff
@@ -90,16 +90,7 @@ class SaveLoginTest: BaseTestCase {
         waitForExistence(app.tables["Login List"])
         XCTAssertTrue(app.staticTexts[domain].exists)
         // XCTAssertTrue(app.staticTexts[domainSecondLogin].exists)
-        if processIsTranslatedStr() == m1Rosetta {
-            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
-        } else {
-            // Change to 3 entries for iPhone since the hostname is not saved and only one entry appears
-            if iPad() {
-                XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
-            } else {
-                XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
-            }
-        }
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
```

Comment:
Would this be the same for M1 and Intel using this new Xcode version?

Reply:
Yes. Same for M1 (iPhone/iPad) and Intel (iPhone/iPad) running XCode 14.3. 🎉

Reply:
This might be the cause of the saveLogin test failure... 
`XCTAssertEqual failed: ("3") is not equal to ("4")`
Don't know why but in some cases there was only 1 login created (different on Intel and M1), and so the count is 3 not 4...